07-github.md

```diff
@@ -140,17 +140,22 @@ Branch master set up to track remote branch master from origin.
 
 > ## Password Managers {.callout}
 >
-> If your operating system has a password manager configured, `git push` will
-> try to use it when it needs your username and password. If you want to type
-> your username and password at the terminal instead of using
-> a password manager, type:
+> Some systems will open a password manager when you run `git push`.  The
+> password manager asks for your github username and password.  Git bash on
+> Windows is the most common example.  If you would prefer to type your
+> username and password at the terminal instead of using the password manager,
+> type:
 >
 > ~~~ {.bash}
 > $ unset SSH_ASKPASS
 > ~~~
 >
-> You may want to add this command at the end of your `~/.bashrc` to make it the
-> default behavior.
+> in the terminal, before you run `git push`.  ([Despite the
+> name](http://git-scm.com/docs/gitcredentials#_requesting_credentials), git
+> uses `SSH_ASKPASS` whether you are using git via SSH or https).
```

Comment:
This addition looks good to me, although I'd suggest shifting the link:

> Despite the name, [Git prefers `SSH_ASKPASS` to terminal prompts for all credential entry][1], not just SSH, so you may want to unset it even if you are using Git over HTTPS.

[1]: http://git-scm.com/docs/gitcredentials#_requesting_credentials